<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dati_spedizione_dialog</class>
 <widget class="QDialog" name="dati_spedizione_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>253</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>673</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Finalizzazione acquisto</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Finalizzazione acquisto</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tutorial">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>87</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Per finalizzare il suo acquisto, inserisca i dati di spedizione, che saranno utili alla spedizione del suo ordine.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_via">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Via:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_via">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_num_civico">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero civico:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txt_num_civico">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cap">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CAP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cap">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_citta">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Citt√†:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_citta">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_finalizza_acquisto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizza acquisto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_annulla_ordine">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>200</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annulla ordine</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
